docs/iii.4.10-ldfld.md

```diff
@@ -10,7 +10,7 @@
 
 #### Description:
 
-The `ldfld` instruction pushes onto the stack the value of a field of _obj_. _obj_ shall be an object (type `O`), a managed pointer (type `&`), an unmanaged pointer (type `native int`), or an instance of a value type. The use of an unmanaged pointer is not permitted in verifiable code. _field_ is a metadata token (a `fieldref` or `fielddef` see [Partition II]#todo-missing-hyperlink)) that shall refer to a field member. The return type is that associated with _field_. `ldfld` pops the object reference off the stack and pushes the value for the field in its place. The field can be either an instance field (in which case _obj_ shall not be null) or a static field.
+The `ldfld` instruction pushes onto the stack the value of a field of _obj_. _obj_ shall be an object (type `O`), a managed pointer (type `&`), an unmanaged pointer (type `native int`), or an instance of a value type. The use of an unmanaged pointer is not permitted in verifiable code. _field_ is a metadata token (a `fieldref` or `fielddef` see [Partition II](ii.22.15-field-0x04.md)) that shall refer to a field member. The return type is that associated with _field_. `ldfld` pops the object reference off the stack and pushes the value for the field in its place. The field can be either an instance field (in which case _obj_ shall not be null) or a static field.
```

Comment:
Btw. the standard contains a few other identical references where the hyperlink is also missing. While we're at it, could you fix those hyperlinks too?

* in [III.4.11](https://github.com/stakx/ecma-335/blob/master/docs/iii.4.11-ldflda.md)
* in [III.4.14](https://github.com/stakx/ecma-335/blob/master/docs/iii.4.14-ldsfld.md)
* in [III.4.15](https://github.com/stakx/ecma-335/blob/master/docs/iii.4.15-ldsflda.md)
* in [III.4.28](https://github.com/stakx/ecma-335/blob/master/docs/iii.4.28-stfld.md)
* in [III.4.30](https://github.com/stakx/ecma-335/blob/master/docs/iii.4.30-stsfld.md)

Reply:
👍  made those changes, and went a little further :D